lib/process-content.js

```diff
@@ -21,6 +21,7 @@ module.exports = function processContent(
   // SugarSS support:
   if (ext === ".sss") {
     if (!sugarss) {
+      /* c8 ignore next 3 */
```

Comment:
We can't check for test coverage for this bit of code because `sugarss` always exists when running the tests here.